src/cc_catalog_airflow/dags/provider_api_scripts/metropolitan_museum_of_art.py

```diff
@@ -0,0 +1,279 @@
+        extra_image_index = 1
+        meta_data['set'] = foreign_url
+
+
+    image_store.add_item(
+        foreign_url,  # foreign url of image
+        image_url,  # image url
+        thumbnail,  # thubnail url
+        None,  # license URl
+        'cc0',  # license
+        '1.0',  # license verion
+        foreign_id,  # foreign identifier
+        None,  # width
+        None,  # height
+        creator_name,  # creator name
+        None,  # creator url
+        title,  # title
+        meta_data,  # meta data
+        None,
+        'f',
+        None
+    )
+
+    for image in other_images:
+        foreign_id = '{}-{}'.format(object_id, extra_image_index)
+        image_url = image
+        thumbnail = ''
+
+        if image_url:
+            if '/original/' in image_url:
+                image_url.replace('/original/', '/web-image/')
+
+        image_store.add_item(
+            foreign_url,  # foreign url of image
+            image_url,  # image url
+            thumbnail,  # thubnail url
+            None,  # license URl
+            'cc0',  # license
+            '1.0',  # license verion
+            foreign_id,  # foreign identifier
+            None,  # width
+            None,  # height
+            creator_name,  # creator name
+            None,  # creator url
+            title,  # title
+            meta_data,  # meta data
+            None,
+            'f',
+            None
+    )
+
+
+
+def _create_meta_data(object_json):
+    meta_data = {}
+
+    meta_data['accession_number'] = object_json.get('accessionNumber', None)
+    meta_data['classification'] = object_json.get('classification', None)
+    meta_data['culture'] = object_json.get('culture', None)
+    meta_data['date'] = object_json.get('objectDate', None)
+    meta_data['medium'] = object_json.get('medium', None)
+    meta_data['credit_line'] = object_json.get('creditLine', None)
+
+    return meta_data
+
+
+if __name__ == '__main__':
+    mode = 'date :'
+    parser = argparse.ArgumentParser(
+        description='Metropolitan Museum of Art API',
+        add_help=True
+    )
+    parser.add_argument(
+        '--date',
+        help='Fetches all the artwork uploaded after given date'
+    )
+    parser.add_argument(
+        '--mode',
+        choices=['default', 'all'],
+        help='Identify all artworks from the previous to previous day [default]'
+         'or process the entire collection [all].'
+    )
+    args = parser.parse_args()
+    if args.date:
+        date = args.date
+
+    elif args.mode:
+        if str(args.mode) == 'default':
+            date_obj = datetime.now() - timedelta(days=2)
+            date = datetime.strftime(date_obj, '%Y-%m-%d')
+        else:
+            date = None
+            mode = 'All CC0 Artworks'
+    else:
+        date_obj = datetime.now() - timedelta(days=2)
+        date = datetime.strftime(date_obj, '%Y-%m-%d')
+
+    mode += date if date is not None else ''
+    logger.info(f'Processing for {mode}')
```

Comment:
We need to ensure that the script acts the same as the main function.  Currently, if I call the script with no `--date` parameter, it runs for 2 days ago.  If I call the `main` function with no `date` parameter, it pulls *all* images from all time from the API.  The latter behavior is the desired one.  Please clean up this if `__name__ == '__main__:' block, and make sure that it behaves as the `main` function does.